crhelper/resource_helper.py

```diff
@@ -96,6 +98,10 @@ def __call__(self, event, context):
             if self._timer:
                 self._timer.cancel()
 
+    def _construct_boto3_client(self, service, **kwargs):
+        botocore_config = config.Config(**self._botocore_config)
+        return boto3.client(service, config=botocore_config, **kwargs)
```

Comment:
Is 
```
boto3.client(service, config={})
```
Exactly the same thing as:
```
boto3.client(service)
```
It's probably safer to only pass `config=` if a boto config has been passed